tests/integration/pox/pox-3/reject-pox.spec.ts

```diff
@@ -0,0 +1,202 @@
+        fee,
+        nonce: bobNonce++,
+      },
+      { amount: 90_000_000_000_010, blockHeight, cycles }
+    );
+    expect(response.error).toBeUndefined();
+    [block, tx] = await waitForStacksTransaction(orchestrator, response.txid);
+    expect(tx.success).toBeTruthy();
+
+    let poxInfo = await getPoxInfo(network);
+    expect(poxInfo.contract_id).toBe("ST000000000000000000002AMW42H.pox-3");
+    expect(poxInfo.current_cycle.id).toBe(1);
+    expect(poxInfo.rejection_votes_left_required).toBeGreaterThan(0);
+
+    await waitForNextRewardPhase(network, orchestrator, 1);
+
+    // Verify PoX is active
+    poxInfo = await getPoxInfo(network);
+    expect(poxInfo.current_cycle.is_pox_active).toBe(true);
+
+    // Chloe rejects pox, but does not have enough to de-activate it
+    response = await broadcastRejectPox({
+      poxVersion: 3,
+      network,
+      account: Accounts.WALLET_3,
+      fee,
+      nonce: chloeNonce++,
+    });
+    expect(response.error).toBeUndefined();
+    [block, tx] = await waitForStacksTransaction(orchestrator, response.txid);
+    expect(tx.success).toBeTruthy();
+
+    let isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.current_cycle.id
+    );
+    expect(isPoxActive).toEqual(trueCV());
+
+    isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.next_cycle.id
+    );
+    expect(isPoxActive).toEqual(trueCV());
+
+    poxInfo = await getPoxInfo(network);
+    expect(poxInfo.rejection_votes_left_required).toBeGreaterThan(0);
+  });
+
+  it("pox should be disabled if enough STX reject", async () => {
+    const network = new StacksTestnet({ url: orchestrator.getStacksNodeUrl() });
+
+    let response = await broadcastRejectPox({
+      poxVersion: 3,
+      network,
+      account: Accounts.FAUCET,
+      fee,
+      nonce: faucetNonce++,
+    });
+    expect(response.error).toBeUndefined();
+    let [block, tx] = await waitForStacksTransaction(
+      orchestrator,
+      response.txid
+    );
+    expect(tx.success).toBeTruthy();
+
+    let poxInfo = await getPoxInfo(network);
+    expect(poxInfo.rejection_votes_left_required).toBe(0);
+
+    let isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.current_cycle.id
+    );
+    expect(isPoxActive).toEqual(trueCV());
+
+    isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.next_cycle.id
+    );
+    expect(isPoxActive).toEqual(falseCV());
+
+    await waitForNextRewardPhase(network, orchestrator);
+
+    poxInfo = await getPoxInfo(network);
+    expect(poxInfo.rejection_votes_left_required).toBe(0);
+
+    isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.current_cycle.id
+    );
+    expect(isPoxActive).toEqual(trueCV());
```

Comment:
Why true here? It was false in the previous cycle for the next cycle

Reply:
Yeah, you're right. I think I made a mistake in the test case here and it accidentally worked. I'll look into it.

Reply:
Ok, I need to go one block more to get the data for the right cycle. I'll push an update soon. Thanks for catching that.

Reply:
Fixed in c32a496.